lib/teleterm/grpccredentials.go

```diff
@@ -118,6 +118,7 @@ func createClientTLSConfig(clientKeyPair tls.Certificate, serverCertPath string)
 
 	return &tls.Config{
 		Certificates: []tls.Certificate{clientKeyPair},
+		NextProtos:   []string{"h2"},
```

Comment:
Is there a const we could use for "h2"?

Could you add some context on why the NextProtos is necessary here too?

Reply:
There's a const in x/net/http2 but it's hardly ever used, even in library code.

I've actually moved this to the test code as well, since we hardly ever manually add "h2" ourselves in the rest of the codebase.